### DISCUSSION TITLE
Unable to compile these release 4.7.0

### DISCUSSION THREAD

**Discussion by sisla-apps** [microsoft-search/pnp-modern-search#2482] [Q&A] [answered]
Need help on compiling the source code locally as im getting the following errors when trying to gulp bundle the solution.
![pnp-modern-search 4 7 0_gulpship-error](https://user-images.githubusercontent.com/58120448/197466152-27322911-e198-467c-be1b-f0e2a385ebcb.png)

I did followed the steps on "Setting up the solution locally" but still get these error.
Can someone please help and point me to the right direction. Apologies as im quite still new to this

Thank you

**Comment by wobba** [comment, marked as answer]
Should be fairly simple. Clone repo, follow SPFx setup guidance (need node v14), run `npm i` and then `gulp`.

**Reply by sisla-apps** [reply]
Thanks so much, i was able to compile it.

**Reply by wobba** [reply]
Great :)
